src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private static final String INDENTION = "\t";
+    private static final String SEPARATOR = ",";
+    private static final String TITLE = "fruits,quantity";
+    private final ReportService reportService = new ReportServiceImpl();
+
+    @AfterEach
+    void tearDown() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    void createReport_emptyStorage_Ok() {
```

Comment:
```suggestion
    void createReport_emptyStorage_ok() {
```
please use lowercase after underscore in test names everywhere